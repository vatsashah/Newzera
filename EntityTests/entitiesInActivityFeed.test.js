// Importing required library
import React from 'react'
import { shallow, mount } from 'enzyme'

// Importing component to be tested
import EntitiesInActivityFeed from '../../../Components/Entity/entitiesInActivityFeed'

describe('Entities In Activity Feed should render without any error', () => {
  it('Entities In Activity Feed should render correctly', () => {
    const component = shallow(<EntitiesInActivityFeed />)
    expect(component).toMatchSnapshot()
  })
  it('Entities In Activity Feed should have same number of entity names and entity images', () => {
    const wrapper = mount(<EntitiesInActivityFeed />).props()
    const numberOfEntityNames = wrapper.followingEntityNames.length
    const numberOfEntityImages = wrapper.followingEntityImages.length
    expect(numberOfEntityImages).toEqual(numberOfEntityNames)
  })
})
