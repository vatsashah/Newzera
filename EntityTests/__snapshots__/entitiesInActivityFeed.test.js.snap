// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Entities In Activity Feed should render without any error Entities In Activity Feed should render correctly 1`] = `
<div>
  <div
    className="EntitiesInActivityFeedSection"
    style={
      Object {
        "left": "0vw",
        "top": "0vh",
      }
    }
  >
    <div
      className="PointerWhenOnHover"
    >
      <img
        alt="User"
        className="UserImage"
        src="userImage.png"
      />
    </div>
    <div
      className="UserContent"
    >
      <div
        className="UserName PointerWhenOnHover"
      >
        Amanda Will 
         
      </div>
        
      <div
        className="MiddleText"
      >
        started following 
      </div>
        
      <div
        className="UserFollowingNames PointerWhenOnHover"
      >
        India, ONU, Donald Trump and 3 more
      </div>
    </div>
    <div
      className="PointerWhenOnHover"
    >
      <img
        alt="elipsis"
        className="elipsis"
        src={
          Object {
            "ReactComponent": Object {
              "$$typeof": Symbol(react.forward_ref),
              "render": [Function],
            },
            "__esModule": true,
            "default": "elipsis.svg",
          }
        }
      />
    </div>
    <div>
      <p
        className="Friends"
      >
        Friends
      </p>
    </div>
    <div
      className="period"
    />
    <div>
      <p
        className="Time"
      >
        4:45 pm
      </p>
    </div>
    <div>
      <div
        key="0"
      >
        <img
          alt="EntityImageInActivityFeed"
          className="EntityImageInActivityFeed"
          src="Trump.png"
          style={
            Object {
              "left": "4.791666666666667vw",
            }
          }
        />
      </div>
      <div
        key="1"
      >
        <img
          alt="EntityImageInActivityFeed"
          className="EntityImageInActivityFeed"
          src="Trump.png"
          style={
            Object {
              "left": "7.916666666666667vw",
            }
          }
        />
      </div>
      <div
        key="2"
      >
        <img
          alt="EntityImageInActivityFeed"
          className="EntityImageInActivityFeed"
          src="Trump.png"
          style={
            Object {
              "left": "11.041666666666668vw",
            }
          }
        />
      </div>
      <div
        key="3"
      >
        <img
          alt="EntityImageInActivityFeed"
          className="EntityImageInActivityFeed"
          src="Trump.png"
          style={
            Object {
              "left": "14.166666666666668vw",
            }
          }
        />
      </div>
      <div
        key="4"
      >
        <img
          alt="EntityImageInActivityFeed"
          className="EntityImageInActivityFeed"
          src="Trump.png"
          style={
            Object {
              "left": "17.291666666666668vw",
            }
          }
        />
      </div>
      <div
        key="5"
      >
        <img
          alt="EntityImageInActivityFeed"
          className="EntityImageInActivityFeed"
          src="Trump.png"
          style={
            Object {
              "left": "20.416666666666668vw",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;
